config/routes.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  # root to 'homepages#index'
+  get '/homepages', to: 'homepages#index', as: 'homepages'
```

Comment:
Just so the homepage goes to the homepages controller

```suggestion
  root to: 'homepages#index'
```